src/actions/trends.jsx

```diff
@@ -107,6 +108,21 @@ export function removeFocus() {
   }
 }
 
+/**
+ * Notifies the application that the tour was toggled
+ *
+ * @param {string} value the new payload from the tooltip
+ * @returns {string} a packaged payload to be used by Redux reducers
+ */
+export function tourToggled( value ) {
```

Comment:
I would write a test for this, but it's unnecessary esp if we are going to convert to redux toolkit.